src/ParkrunMap.UI/ParkrunMap.UI/ClientApp/src/components/ParkrunMap.js

```diff
@@ -33,10 +34,34 @@ function mapDispatchToProps(dispatch) {
 }
 
 function mapStateToProps(state) {
-  return {
+
+  const props = {
     location: state.location.location,
     parkruns: state.parkruns.parkruns
+  };
+
+  if (state.routing.location.pathname !== '/') {
+
+    var parkruns = state.parkruns.parkruns.filter(parkrun => {
+      const filters = {
+        wheelchairFriendly: state.routing.location.pathname === '/wheelchair-friendly',
+        buggyFriendly: state.routing.location.pathname === '/buggy-friendly'
+      };
+      var selected = false;
```

Comment:
No `var`! Use let!

But you could make this easier to maintain, something like:

const selectedWhen = [
  filters.wheelchairFriendly && parkrun.features.wheelchairFriendly,
  filters.buggyFriendly && parkrun.features.buggyFriendly,
];

return selectedWhen.some(x => x);
```